pkg/utils/utils_test.go

```diff
@@ -67,4 +69,16 @@ var _ = Describe("Utils", func() {
 			Expect(err).To(HaveOccurred(), "Not existing VF should return an error")
 		})
 	})
+	Context("Checking GetGUIDFromHwAddr function", func() {
+		It("Valid IPoIB hardware address", func() {
+			hwAddr, _ := net.ParseMAC("0000.0000.fe80.0000.0000.0000.0200.5e10.0000.0001")
```

Comment:
any reason you chose a different input format for net.ParseMac() then in other test files ?

Reply:
There was no any strict reason, besides of less typing.